src/main/resources/static/css/home-page-style.css

```diff
@@ -13,14 +13,18 @@ h2 {
 .button {
     background-color: var(--daeker-blue);
     border: none;
-    color: white;
-    padding: 15px 50px;
     text-align: center;
     text-decoration: none;
     display: inline-block;
     font-size: 16px;
     width: 85%;
     gap: min(2px, 1%);
+    font-weight: 400;
+    padding: .375rem .75rem;
+	line-height: 1.5;
+	border-radius: .25rem;
+	color: #fff;
+	background-color: #007bff;
```

Comment:
Could we make sure that our formatting is consistent?

(If you merge main into here and run `./gradlew spotlessCheck` then `./gradlew spotlessApply`, that should format things automatically for you)

Reply:
done

Reply:
than you